Dockerfile

```diff
@@ -1,3 +1,15 @@
 FROM nginx:alpine
-COPY ./www /usr/share/nginx/html
+
+# install required packages
+RUN apk add --no-cache git
+RUN apk add --update nodejs npm
+
+# Copy the contents and build 
+RUN mkdir /tmp/source
+COPY ./ /tmp/source
+WORKDIR /tmp/source
+RUN npm install
+RUN node_modules/gulp/bin/gulp.js build-min
+
+RUN cp -r /tmp/source/www /usr/share/nginx/html
```

Comment:
Should this be a cp, or a mv?
Just thinking we're going to be leaving it in /tmp/source/ which seems wrong.